internal/udp/conn.go

```diff
@@ -41,41 +44,25 @@ func (l *Listener) Accept() (*Conn, error) {
 
 // Close closes the listener.
 // Any blocked Accept operations will be unblocked and return errors.
-func (l *Listener) Close(shutdownTimeout time.Duration) error {
-	l.lock.Lock()
-	defer l.lock.Unlock()
-
-	var err error
+func (l *Listener) Close() error {
+	err := errClosedListener
```

Comment:
Our dtls implementation calls `Listener.Close()` multiple times.
It should allow multiple calls for now.